### ISSUE TITLE
Speed to high at tight turns

### ISSUE THREAD

**Issue by hpz937** [Stephan-S/FS19_AutoDrive#68]
I have notice that my vehicles are taking turns way too fast, sometimes nearly full speed at 90 degree turns. this cause the ai to overshoot the track then overcorrect to get back causing the ai to drive in circles or get stuck on obstacles.

It is typically the faster vehicles that this happens to semi-trucks, faster tractors, but not always. I am experiencing this issue on 1.0.1.1, I switched back to 1.0.0.9 and have not had this happen to me yet.

**Comment by Stephan-S** [comment]
Hey, yeah I was trying to increase the speed a bit. But I also introduced the lookahead braking value in the settings menu. So make sure this value is set accordingly. e.g. around 25m

**Comment by hpz937** [comment]
I set the look ahead to 25 and it did not seem to make much difference. I ran the course several time and what seems to be happening is, the tractor slows down quite a bit right before making the turn to about 12kph then it starts making the turn then accelerates to 40kph while still making the turn then the tractor can't make it to the path and drives in circles.

I ended up fixing the issue on a couple really bad intersections by just re-recording the path to be much wider and it seems to follow it now. I do end up driving down the wrong side of the road for a bit to accomplish this, although this would be better for larger vehicles that I have not tested this with yet..
